### PR TITLE
README

### PR DESCRIPTION
**Pull request by billdybas**
Adds a top level README about our project, team, architecture.

- [x] Only section I have left is the "Process" section.

Let me know if there's something I missed we should add or if anything should be re-worded / changed / removed.

When we add in more tools (eg. image processing) we can add them then. And when we deploy, we can write those instructions.

- [x] Probably should also add a general "Here's how to maintain Node" section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- [GraphQL](http://graphql.org/)
+- [Apollo GraphQL](https://www.apollographql.com/)
+- [Apollo Developers Blog](https://dev-blog.apollodata.com/)
+- [The GitHub GraphQL API](https://githubengineering.com/the-github-graphql-api/)
+- [From REST to GraphQL](https://0x2a.sh/from-rest-to-graphql-b4e95e94c26b) by Jacob Gillespie
+- [awesome-graphql](https://github.com/chentsulin/awesome-graphql)
+
+## Development
+
+You'll need to be running both the frontend and backend for development. Check out each of their README's for further instructions.
```
Comment (chtinahow) [inline README.md]: I would probably reiterate the instructions here or link to the front end read me and then backend readme

Comment (kayladavis) [reply README.md]: I'd lean towards linking.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Both the frontend and backend are written in JavaScript.
+
+Frontend:
+
+- [React](https://reactjs.org/) for view rendering
+- [Redux](https://redux.js.org/) for state management
+- [React Router](https://reacttraining.com/react-router/) v4 for page routing
+- [Apollo Client](https://github.com/apollographql/apollo-client) for GraphQL querying and data caching
+- [Boostrap](http://getbootstrap.com/) v4 (via [reactstrap](https://reactstrap.github.io/)) & [styled-components](https://www.styled-components.com/) for styling
+- [Formik](https://github.com/jaredpalmer/formik) w/ [yup](https://github.com/jquense/yup) for forms
```
Comment (robmcl4) [inline README.md]: mention webpack somehow... seems a biggie

Comment (billdybas) [reply README.md]: Oh, yes, definitely. Slipped my mind completely.

File: README.md (hunk 1 above)
Comment (billdybas) [reply README.md]: I'm going to say neither. You're in the top level of the repo, it's not hard to click on the  `backend` or `frontend` folders and read their README. Additionally, links inside our own repo will require maintenance – eg. when we move the repo to the org, we'll have to update the links.

**Pull request merged**